docs/3.x.x/guides/deployment.md

```diff
@@ -75,3 +83,317 @@ We highly recommend to use [pm2](https://github.com/Unitech/pm2/) to manage your
 ### Advanced configurations
 
 If you want to host the administration on another server than the API, [please take a look at this dedicated section](../advanced/customize-admin.md#deployment).
+
+## Heroku
```

Comment:
Should fair warn the user that you cannot use the content-type builder with heroku at all.

Reply:
@pierreburgy  added something at the end of deployment.md regarding this.

Reply:
The content type builder is disabled in production anyway.